container/openejb-jpa-integration/src/main/java/org/apache/openejb/jpa/integration/eclipselink/MBeanOpenEJBRuntimeServices.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.openejb.jpa.integration.eclipselink;
+
+import org.eclipse.persistence.internal.sessions.AbstractSession;
+import org.eclipse.persistence.sessions.Session;
+
+public class MBeanOpenEJBRuntimeServices extends OpenEJBRuntimeServices implements MBeanOpenEJBRuntimeServicesMBean {
```

Comment:
ideally we would have a single impl, cant we reuse one existing impl since they do nothing here?

only PLATFORM_NAME is set and it is not really needed since only an error thing

Reply:
I certainly can use the Glassfish one here if you would like. But, to remain consistent with how all the other ones are implemented, I would suggest we don't.

![screen shot 2018-10-08 at 1 18 15 pm](https://user-images.githubusercontent.com/1392297/46626331-a5618080-cafc-11e8-8f13-72905a7debe2.png)

Reply:
Was more thinking about dropping this and keeping only OpenEJBRuntimeServices